packages/react-cli/src/utils/shell.ts

```diff
@@ -61,6 +60,17 @@ export async function doesProgramExist(name: string) {
   }
 }
 
+export async function doesApplicationHaveModule(name: string): Promise<Boolean>{
+  try {
+    const moduleList = await exec(`npm ls --parseable --prod`);
```

Comment:
you probably want to read in the package.json and check `dependencies` specifically.